src/stories/components/form.js

```diff
@@ -174,18 +174,20 @@ storiesOf('Form', module)
                 },
                 null
             );
-            const disabled = boolean('disabled', false);
+            const isDisabled = boolean('Disabled', false);
```

Comment:
Now the disabled state doesn't work.
This const should stay `disabled` not `isDisabled`, because ReactSwitch takes prop `disabled`

Reply:
yup, I somehow messed up resolving conflicts after the merge 🤷‍♀️ fixed now